models/orderDao.js

```diff
@@ -0,0 +1,36 @@
+const { myDataSource } = require("./myDataSource");
+
+// 주문/결제 내역 조회
+const read = async (userId) => {
```

Comment:
데이터를 가져오는 함수명에 get도 많이 사용하니 참고해 주세요.
```suggestion
const readOrderList = async (userId) => {
```
or
```suggestion
const getOrderList = async (userId) => {
```